wiki/markdown/docker.md

```diff
@@ -0,0 +1,68 @@
+# Docker
+
+In Eiffel-Intelligence frontend source code a Dockerfile is provided which helps the developer or user to build the local Eiffel-Intellegence frontend source code changes to a Docker image.
+With the Docker image user can tryout the Eiffel-Intelligence frontend on a Docker Host or in a Kubernetes cluster.
+
+## Requirements
+- Docker 
+
+
+  Linux: https://docs.docker.com/install/linux/docker-ce/ubuntu/
+
+  
+  Windows: https://docs.docker.com/docker-for-windows/install/
+
+## Follow these step to build the Docker image.
+
+1. Build the Eiffel-Intelligence frontend war file: 
+`mvn package -DskipTests` 
+
+
+This will produce a war file in the "target" folder.
+2. Build the Docker image with the war file that was produced from previous step: 
```

Comment:
The second instruction doesn't come on a new line because markdown interprets it as connected with the above sentence.

Reply:
Fixed.